src/code/z_fbdemo_fade.c

```diff
@@ -96,12 +96,18 @@ void TransitionFade_Update(void* thisx, s32 updateRate) {
     }
 }
 
+#if PLATFORM_N64
+#define ALPHA_NOT_ZERO color->a != 0
+#else
+#define ALPHA_NOT_ZERO color->a > 0
+#endif
+
 void TransitionFade_Draw(void* thisx, Gfx** gfxP) {
     TransitionFade* this = (TransitionFade*)thisx;
     Gfx* gfx;
     Color_RGBA8_u32* color = &this->color;
 
-    if (color->a > 0) {
+    if (ALPHA_NOT_ZERO) {
```

Comment:
hm that's annoying, thanks for looking into this. I wonder if it would be easier to read as this?

```c
    if (
#if PLATFORM_N64
        color->a != 0
#else
        color->a > 0
#endif
    ) {
        ....
    }
```

I'll let some other reviewers weigh in

Reply:
my first thought was something like:
```c
#if PLATFORM_N64
if (color->a != 0)
#else
if (color->a > 0)
#endif
{
    ....
}
```
but then I thought a define would be cleaner

Reply:
of these two suggestions, I think yanis' is easier to read. but define is also fine i guess

Reply:
I find the define harder to read since it breaks up the flow having to find the definition of the macro, even though it superficially looks cleaner. I'm good with yanis' suggestion too